inc/form.class.php

```diff
@@ -690,6 +690,11 @@ public function showWizard() : void {
    public static function getFormList(int $rootCategory = 0, string $keywords = '', bool $helpdeskHome = false): array {
       global $DB, $TRANSLATE;
 
+      $display_format = PluginFormcreatorEntityconfig::CONFIG_UI_FORM_MASONRY;
+      if (PluginFormcreatorEntityconfig::getUsedConfig('tile_design', Session::getActiveEntity()) == PluginFormcreatorEntityconfig::CONFIG_UI_FORM_UNIFORM_HEIGHT) {
+         $display_format = PluginFormcreatorEntityconfig::CONFIG_UI_FORM_UNIFORM_HEIGHT;
+      }
```

Comment:
Why not simply write the following ? If the entity configrow does not exists, it is created by getUsedConfig().

```suggestion
      $display_format = PluginFormcreatorEntityconfig::getUsedConfig('tile_design', Session::getActiveEntity());
```